deploy/ocp/values.yaml

```diff
@@ -3,7 +3,7 @@ namespace: openshift-operator-lifecycle-manager
 catalog_namespace: openshift-operator-lifecycle-manager
 operator_namespace: openshift-operators
 imagestream: true
-writeStatusName: operator-lifecycle-manager
```

Comment:
I think this is the only values file where we want `writeStatusName: operator-lifecycle-manager` set.